.circleci/config.yml

```diff
@@ -64,10 +64,12 @@ commands:
             - tox_cache-{{ checksum "tox.ini" }}
       - run:
           name: pyenv setup
-          command: pyenv local 2.7.16 3.5.7 3.6.8 3.7.12 3.8.12 3.9.10 3.10.2
+          command: pyenv local 3.8.19 3.9.19 3.10.14 3.11.8 3.12.2
       - run:
           name: tox
           command: tox --pre
+          environment:
+            VIRTUALENV_DISCOVERY: pyenv
```

Comment:
this environment variable is needed to migrate off of tox-pyenv per instructions here:

https://github.com/tox-dev/tox-pyenv/tree/master?tab=readme-ov-file#tox-pyenv